appsec/src/extension/tags.c

```diff
@@ -1423,6 +1433,110 @@ static PHP_FUNCTION(datadog_appsec_track_user_login_failure_event)
     dd_tags_set_sampling_priority();
 }
 
+static PHP_FUNCTION(datadog_appsec_track_authenticated_user_event_automated)
```

Comment:
we have a `user_tracking.c` file. We shouldn't shove any functionality involving tags (regardless of the particular feature) into this file